### PR TITLE
Add documentation for backoff delay

### PR DESCRIPTION
**Pull request by alenkacz**
JIRA issues: MARATHON-8506

### CODE REVIEW

File: docs/_layouts/docs.html [hunk 1]
```diff
@@ -228,6 +228,11 @@ <h5>Troubleshooting</h5>
                    Stuck App or Pod Deployments
                 </a>
             </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/backoff.html">
+                    No new instances are starting
```
Comment (kensipe) [inline docs/_layouts/docs.html]: this is troubleshooting... my concern with putting a link specifically for backoff delay for "No new instances are starting" is that there could be multiple reasons why no new instances are starting and this is one of the reasons.   Wouldn't it make more sense to have a list for No new instances are starting of reasons why with a link to each.. which includes this one?

File: docs/docs/backoff.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+title: No new instances of app are starting, it is in 'delayed' mode
+---
+
+# No new instances of app are starting, it is in 'delayed' mode
+
+One of the reasons why is your deployment not finishing and your instances appear to start very slowly is that your application 
+might be in 'delayed' mode.
```
Comment (kensipe) [inline docs/docs/backoff.md]: there are many line breaks throughout.. these should just be one line IMO

File: docs/docs/backoff.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## When is the delay reset to default
+
+- when new version of an application is deployed
+- when `DELETE /v2/queue/{your_application_name}/delay` HTTP API request is issued
+- when there are no failing tasks for the value of current delay + maxLaunchDelay
+
+## How do I reset the delay?
+
+By issuing a request to `DELETE /v2/queue/{your_application_name}/delay`.
```
Comment (kensipe) [inline docs/docs/backoff.md]: `your_application_name` is `app-id`

**Comment by kensipe** [comment]
updates pushed

File: docs/docs/backoff.md (hunk 3 above)
Comment (timcharper) [reply docs/docs/backoff.md]: service id :) I fixed it

File: docs/docs/backoff.md (hunk 2 above)
Comment (timcharper) [reply docs/docs/backoff.md]: I submit we should do a sentence per line as our general standard; I really like how well diffs render.

**Pull request merged**